demisto_sdk/commands/common/content/content.py

```diff
@@ -127,3 +128,143 @@ def content_descriptor(self) -> Optional[ContentDescriptor]:
             descriptor_object = ContentDescriptor(path)
 
         return descriptor_object
+
+    def modified_packs(self, prev_ver='master', committed_only=False, staged_only=False) -> Dict[str, Pack]:
+        content_repo: Repo = self.git()
+
+        # staging all local changes
+        content_repo.git.add('.')
+
+        committed_set = {Path(os.path.join(*Path(item.a_path).parts[:Path(item.a_path).parts.index('Packs') + 2]))
+                         for item in content_repo.remote().refs[prev_ver].commit.diff(
+            content_repo.active_branch).iter_change_type('M')}
+
+        committed: Dict = {path.name: Pack(path) for path in committed_set}
+
+        if committed_only:
+            return committed
+
+        staged_set = {Path(os.path.join(*Path(item.a_path).parts[:Path(item.a_path).parts.index('Packs') + 2])) for item
+                      in content_repo.head.commit.diff(
+            paths=list((Path().cwd() / 'Packs').glob('*/'))).iter_change_type('M')}
```

Comment:
we should also mark renamed files as modified when the rename is not 100%

not sure if this implementation collects them.